### PR TITLE
docs: fixes from docbash v2

### PR DESCRIPTION
**Pull request by frederik-encord**
1. Remove unnecessary content from FAQ
2. Rerun build script for metrics to get in latest updates

### CODE REVIEW

File: docs/docs/metrics/heuristic.md [hunk 1]
```diff
@@ -112,19 +112,19 @@ track-ids, they will be flagged as potential inconsistencies in tracks.
...
+Implementation on [GitHub](https://github.com/encord-team/encord-active/blob/main/src/encord_active/lib/metrics/heuristic/high_iou_changing_classes.py)
 
+## Missing Objects and Broken Tracks  
 Identifies missing objects by comparing object overlaps based
 on a running window.
 
 **Case 1:**
 If an intermediate frame (frame $i$) doesn't include an object in the same
-region as in the two surrounding frames ($i-1$ and $i+1$), it is flagged.
+region, as the two surrounding framge ($i-1$ and $i+1$), it is flagged.
```
Comment (eloy-encord) [inline docs/docs/metrics/heuristic.md:127]:
```suggestion
region, as the two surrounding frames ($i-1$ and $i+1$), it is flagged.
```

File: docs/docs/metrics/heuristic.md [hunk 2]
```diff
@@ -159,26 +158,34 @@ hash, they will be flagged as a potentially broken track.
...
-
-Counts number of objects in the image.
+## Object Count  
+Counts number of objects in the image.  
 
 Implementation on [GitHub](https://github.com/encord-team/encord-active/blob/main/src/encord_active/lib/metrics/heuristic/object_counting.py)
 
-## Red Values
+## Random Values on Images  
+Uses a uniform distribution to generate a value between 0 and 1 to each image
```
Comment (eloy-encord) [inline docs/docs/metrics/heuristic.md:172]:
```suggestion
Uses an uniform distribution to generate a value between 0 and 1 to each image  
```

File: docs/docs/metrics/heuristic.md [hunk 3]
```diff
@@ -159,26 +158,34 @@ hash, they will be flagged as a potentially broken track.
...
-## Red Values
+## Random Values on Images  
+Uses a uniform distribution to generate a value between 0 and 1 to each image  
 
-This metric ranks images by how much red is present in the image.
+Implementation on [GitHub](https://github.com/encord-team/encord-active/blob/main/src/encord_active/lib/metrics/heuristic/random.py)
 
-Implementation on [GitHub](https://github.com/encord-team/encord-active/blob/main/src/encord_active/lib/metrics/heuristic/img_features.py)
+## Random Values on Objects  
+Uses a uniform distribution to generate a value between 0 and 1 to each object
```
Comment (eloy-encord) [inline docs/docs/metrics/heuristic.md:177]:
```suggestion
Uses an uniform distribution to generate a value between 0 and 1 to each object  
```

File: docs/docs/metrics/heuristic.md (hunk 2 above)
Comment (frederik-encord) [reply docs/docs/metrics/heuristic.md:172]: these are actually grammatically correct.

File: docs/docs/metrics/heuristic.md (hunk 3 above)
Comment (frederik-encord) [reply docs/docs/metrics/heuristic.md:177]: these are actually grammatically correct.

**Review by eloy-encord — APPROVED** [review]

**Pull request merged**